Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -52,6 +52,7 @@
 #include "SharedBufferReference.h"
 #include "UserData.h"
 #include "WebColorChooser.h"
+#include "WebCore/LoginStatus.h"
```

Comment:
When including from another namespace the include should have `<>` instead of `""`.